ci/test/00_setup_env_mac_host.sh

```diff
@@ -6,6 +6,7 @@
 
 export LC_ALL=C.UTF-8
 
+export DOCKER_NAME=native_macos
```

Comment:
Note that this line is without affect, as the native macOS native build doesn't use Docker (more detailled explanation: the script of this build sets `export RUN_CI_ON_HOST=true` a few lines below, and the install script `04_install.sh` only starts Docker if `RUN_CI_ON_HOST` is _not_ set.)

Reply:
Thanks, I've fixed this up.